app.js

```diff
@@ -11,7 +11,7 @@ const app = express();
 
 // specified on which URL we allow cors
 const corsOptions = {
-  origin: process.env.FRONTEND_URL,
+  origin: true,
   credentials: true,
```

Comment:
change in CORS options thanks to Camille. For the moment we don't have explanation WHY but it works!!